src/nativescript/user/user.android.js

```diff
@@ -0,0 +1,36 @@
+import { User as CoreUser } from '../../core/user';
+
+export class User extends CoreUser {
```

Comment:
Looks suspicious. Intentionally empty?

Reply:
This is no longer needed. I will remove it.